WISE-Community/WISE
# Merged ChangeStudentPeriodController into ManageStudentsController

How to test:
1. Log in as a teacher and make sure that you can change the period of the students.
2. Also make sure that other functions in manage students controller (viewMyStudents, getStudentList, exportStudentList, handleWorkgroupChanges) still work.

Closes #1037.
